python/lsst/faro/measurement/ForcedSourceTableMeasurement.py

```diff
@@ -0,0 +1,165 @@
+# This file is part of faro.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+import lsst.pipe.base as pipeBase
+import lsst.pex.config as pexConfig
+
+from lsst.faro.base.CatalogMeasurementBase import (
+    CatalogMeasurementBaseConnections,
+    CatalogMeasurementBaseConfig,
+    CatalogMeasurementBaseTask,
+)
+
+__all__ = (
+    "ForcedSourceTableMeasurementConnections",
+    "ForcedSourceTableMeasurementConfig",
+    "ForcedSourceTableMeasurementTask",
+    "ForcedSourceMultiBandTableMeasurementConnections",
+    "ForcedSourceMultiBandTableMeasurementConfig",
+    "ForcedSourceMultiBandTableMeasurementTask",
+)
+
+
+class ForcedSourceTableMeasurementConnections(
+    CatalogMeasurementBaseConnections,
+    dimensions=("tract", "skymap", "band"),
+):
+
+    catalog = pipeBase.connectionTypes.Input(
+        doc="Forced source table in parquet format, per tract",
+        dimensions=("tract", "skymap"),
+        storageClass="DataFrame",
+        name="forcedSourceTable_tract",
+        deferLoad=True,
+    )
+
+    measurement = pipeBase.connectionTypes.Output(
+        doc="Per-tract measurement.",
+        dimensions=("tract", "skymap", "band"),
+        storageClass="MetricValue",
+        name="metricvalue_{package}_{metric}",
+    )
+
+
+class ForcedSourceTableMeasurementConfig(
+    CatalogMeasurementBaseConfig, pipelineConnections=ForcedSourceTableMeasurementConnections
+):
+    """Configuration for ForcedSourceTableMeasurementTask."""
+
+    columns = pexConfig.ListField(
+        doc="Band-independent columns from forcedSourceTable_tract to load.",
+        dtype=str,
+        default=["coord_ra", "coord_dec", "band", "detect_isPrimary", "psfFlux", "psfFluxErr"],
+    )
+
+    instrument = pexConfig.Field(
```

Comment:
What is this `instrument` configuration parameter used for? I imagine that it could be used for computing color terms when comparing to an external reference catalog.

Reply:
Oops - that was residual from copying over from other base classes (leftover from reference cats, indeed). Good catch.